backend/src/api/users/user.model.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const UserSchema = new Schema({
+  username: { type: String, required: true },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+  },
+  passwordHash: { type: String },
+  balance: { type: Number, default: 0, required: true },
+  flatPrice: { type: Number, default: 0, required: true },
+  flatSquareMeters: { type: Number, default: 0, required: true },
+  totalSalary: { type: Number, default: 0, required: true },
+  passiveIncome: { type: Number, default: 0, required: true },
+  incomePercentageToSavings: { type: Number, default: 0, required: true },
+  giftsUnpacked: { type: Number, default: 0 },
+  giftsForUnpacking: { type: Number, default: 0 },
+  tokens: [{ type: String }],
+  createdAt: {
```

Comment:
You could pass second parameter to `Schema` constructor with `{ timestamps: true }`. It would manage createdAt, updatedAt logic by it own (so you don't have to write them in Schema even)

Reply:
Thanks for the notice - fixed!